src/utils/helper.js

```diff
@@ -0,0 +1,3 @@
+export const isDemoMode = search => {
+  return new URLSearchParams(search).get('demo') === 'true'
```

Comment:
I like it that you put it as a helper!
Maybe we can make it case-insensitive, so it will accept `demo=TRUE`, `demo=TrUe`, etc.
Here are my 3 suggestions for how to achieve the same goal, just different code styling:
(consider `URLSearchParams.get` returns `null` if a parameter by the provided name does not exist)
```suggestion
  return new URLSearchParams(search).get('demo')?.toLowerCase?.() === 'true'
```
```suggestion
  return (new URLSearchParams(search).get('demo') ?? '').toLowerCase() === 'true'
```
```suggestion
  const demoParam = new URLSearchParams(search).get('demo')
  return demoParam ? demoParam.toLowerCase() === 'true' : false
```